specification/translation/Azure.AI.DocumentTranslation/main.tsp

```diff
@@ -0,0 +1,150 @@
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+
+@useAuth(ApiKeyAuth<ApiKeyLocation.header, "Ocp-Apim-Subscription-Key">)
+@service({
+  title: "Azure.AI.DocumentTranslation",
+})
+@versioned(DocumentTranslation.Versions)
+namespace DocumentTranslation;
+
+/** The Azure.AI.DocumentTranslation versions. */
+enum Versions {
+  /** The 2023-11-01-preview version of the Azure.AI.DocumentTranslation. */
+  @useDependency(Azure.Core.Versions.v1_0_Preview_1)
+  `2023-11-01-preview`,
+}
+
+alias CommonParameters = {
+  /** Mandatory API version parameter */
+  @query("api-version")
+  apiVersion: string;
```

Comment:
nit, suggest to use Azure.Core model via e.g. `...ApiVersionParameter`
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/foundations.tsp#L131-L137